### PR TITLE
bind host tags to envvar and document them

### PR DESCRIPTION
**Pull request by xvello**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: Dockerfiles/agent/README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ The following environment variables are supported:
 
 - `DD_API_KEY`: your API key (**required**)
 - `DD_HOSTNAME`: hostname to use for metrics
+- `DD_TAGS`: host tags, separated by spaces. For example: `simple-tag-0 tag-key-1:tag-value-1`
```
Comment (olivielpeau) [inline Dockerfiles/agent/README.md]: what do you think about documenting this change in [docs/agent/config.md](https://github.com/DataDog/datadog-agent/blob/6.0.0-beta.8/docs/agent/config.md) also?

File: releasenotes/notes/host-tags-envvar-993e5819ac435cfb.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    The DD_TAGS environment variable allows to set host tags, in addition
+    to the `tags` option in datadog.yaml
+upgrade:
+  - |
+    Tags in the DD_TAGS environment variable are now separated by spaces
+    instead of comas in agent5
```
Comment (olivielpeau) [inline releasenotes/notes/host-tags-envvar-993e5819ac435cfb.yaml]: typo: `commas`

**Review by olivielpeau — CHANGES_REQUESTED** [review]
a couple of nits, other than that LGTM! 👍

File: Dockerfiles/agent/README.md (hunk 1 above)
Comment (xvello) [reply Dockerfiles/agent/README.md]: The space-separation is only for envvars, `datadog.yaml` keeps the yaml-array format. This is due to how viper parses stringarrays in envvars

**Review by olivielpeau — APPROVED** [review]

**Pull request merged**